internal/daysteps/daysteps.go

```diff
@@ -1,13 +1,57 @@
 package daysteps
 
+import (
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/Yandex-Practicum/tracker/internal/personaldata"
+	"github.com/Yandex-Practicum/tracker/internal/spentenergy"
+)
+
 type DaySteps struct {
-	// TODO: добавить поля
+	Steps    int
+	Duration time.Duration
+	Personal personaldata.Personal
```

Comment:
Вот так сделай и все заработает
```suggestion
	personaldata.Personal
```

Reply:
все переделал но в ветке second. проходит все тесты

Reply:
нужно сдавать всю работу в одной ветке в одном пуллреквесте

Reply:
Перекопировать из второй ветки?